src/rdb_protocol/datum_stream.cc

```diff
@@ -1991,11 +1991,13 @@ std::vector<changespec_t> union_datum_stream_t::get_changespecs() {
 range_datum_stream_t::range_datum_stream_t(bool _is_infinite_range,
                                            int64_t _start,
                                            int64_t _stop,
+                                           int64_t _step,
                                            backtrace_id_t bt)
     : eager_datum_stream_t(bt),
       is_infinite_range(_is_infinite_range),
```

Comment:
Oh I see you're checking for 0 below in the range_term_t. I wonder if we should allow `0`. It could be useful if you want an infinite sequence of one number. Either way should be fine.